src/Utilities/ArchiveTools.ts

```diff
@@ -0,0 +1,40 @@
+import { exec } from "@actions/exec";
+import { which } from "@actions/io";
+import PowerShellToolRunner from "./PowerShellToolRunner";
+
+export class ArchiveTools {
+    private use7Zip: boolean;
+
+    constructor(use7Zip = false) {
+        this.use7Zip = use7Zip;
+    }
+
+    public async unzip(zipPath: string, destination: string) {
+        if (this.use7Zip) {
+            await this.unzipUsing7Zip(zipPath, destination);
+        } else {
+            await this.unzipUsingPowerShell(zipPath, destination)
+        }
+    }
+
+    private async unzipUsing7Zip(zipPath: string, destination: string) {
+        const path7Zip = await which("7z.exe", true);
```

Comment:
add a log statement saying we are using 7zip for extraction.

Reply:
added debug log